### PR TITLE
Adds additional readings for unitbonus1

### PR DESCRIPTION
**Pull request by josejuanmartinez**
Adds 3 links Juan Martinez' series of medium posts about the historical perspective of AI in videogames

### CODE REVIEW

**Comment by simoninithomas** [comment]
Hi, thanks for the addition, to merge it you also need to:

Rename the file to additional-readings.mdx

 modify _toctree.yml and add to the correct unit

  - local: unitbonus1/additional-readings
    title: Additional Readings

File: units/en/unitbonus1/ai-additional-readings.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Additional readings about the history of AI in Video Games
+
+If you are interested in knowing more about how AI in video games evolved from the 50s into nowaydays, to have a historical perspective,
```
Comment (Ivan-267) [inline units/en/unitbonus1/ai-additional-readings.mdx:3]:
```suggestion
If you are interested in knowing more about how AI in video games evolved from the 50s into nowadays, to have a historical perspective,
```

**Review by Ivan-267 — APPROVED** [review]
Thanks for correcting the typo, @simoninithomas will need to check the rest.

**Comment by simoninithomas** [comment]
Thanks I checking if it passes the test, thanks @Ivan-267 for the update too 🤗

**Comment by simoninithomas** [comment]
LGTM I merge it 🔥

**Pull request merged**